py/torch_tensorrt/fx/converters/nn_ops_converters.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+# @manual=//deeplearning/trt/python:py_tensorrt
+import tensorrt as trt
+import torch
+
+from ..converter_registry import tensorrt_converter
+
+from .converter_utils import mark_as_int8_layer
+import activation
+
+
+@tensorrt_converter(torch.nn.functional.relu)
+@tensorrt_converter(torch.nn.modules.activation.ReLU)
```

Comment:
The idea was to separate out the torch.nn, torch.ops.aten and torch.nn into separate files and register them in separate files. Then the common implementation would be in operator.py and activation.py. So they are reusing the existing acc_ops implementation in many cases, just that it is placed in operator.py and activation.py.
At present we do not need these converters perse, and since these do not have tests, I will remove them.